### PR TITLE
feat(eek): new eek icons

### PR DESCRIPTION
**Pull request by agliga**
## Description
Added new component eek

## Context
~~Had to add two new icons, one for the up arrow to specify the range, and one for the end arrow (the rounded part)
On safari had to override some styles which were not working properly.~~ 
Added one new icon for arrow. Rest is using CSS
I had to add a container to get it to color correctly and align correctly not using position absolute. 

## References
#1374

## Screenshots
Screen Shot 2021-03-29 at 11 33 47 AM

### CODE REVIEW

**Review by austingardner — APPROVED** [review]
LGTM!

File: docs/_includes/common/eek.html [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+                    </span>
+                </td>
+
+            </tr>
+            {% endfor %}
+        </tbody>
+    </table>
+
+    <p>
+        And for each range, making sure each rating exists within that given range, the EEK can have the following
```
Comment (ianmcburnie) [inline docs/_includes/common/eek.html:40]: Is the end of this sentence correct?

**Review by ianmcburnie — APPROVED** [review]

File: docs/_includes/common/eek.html (hunk 1 above)
Comment (agliga) [reply docs/_includes/common/eek.html:40]: Yes, because it leads into the table that follows.

Comment (ianmcburnie) [reply docs/_includes/common/eek.html:40]: Okay, seems a bit odd though to end in a `:` then have a paragraph in between the table.

**Pull request merged**